### DISCUSSION TITLE
Upgrade Apache 2.4.29-1 to 2.4.52 on frePPLe 6.19 due to CVE-2021-44224/CVE-2021-44790 vulnerabilities

### DISCUSSION THREAD

**Discussion by sgfisme** [frePPLe/frepple#516] [Q&A] [answered]
Due to CVE-2021-44224 and CVE-2021-44790 vulnerabilities, could we upgrade Apache web server from 2.4.29-1 to 2.4.52 on frepple 6.19 by ourselves?

**Comment by jdetaeye** [comment, marked as answer]
When we build the frepple image it is based on the latest updated ubuntu docker image (https://github.com/docker-library/docs/commits/master/ubuntu).  We count on that base image to have the latest security updates.

Between frepple releases we are not rebuilding the frepple container with every update.  You will have to rebuild the docker image yourself - the steps are completely transparent on https://github.com/frePPLe/frepple/blob/master/.github/workflows/ubuntu18.yml
